curriculum/challenges/english/25-front-end-development/lecture-best-practices-for-responsive-web-design/672cf07a2b9796eb49719e03.md

```diff
@@ -50,35 +50,35 @@ Think about how websites adapt to various device sizes.
 
 ## --text--
 
-In a mobile-first approach, which type of media query is typically used for breakpoints?
+Which of the following is a commonly used breakpoint for smaller devices like smart phones?
 
 ## --answers--
 
-`max-width`
+1640px
 
 ### --feedback--
 
-Consider how styles are added as screen sizes increase from mobile to desktop.
+Review the beginning of the video to see where this was discussed. 
 
 ---
 
-`min-width`
+up to 640px
```

Comment:
maybe this should be
```suggestion
640px
```

having an option different from the others is suspicious for whom answer the question and may try that without thinking about the answer